README.md

```diff
@@ -8,19 +8,48 @@ Our open source community's Wiki focuses on answering your questions about learn
 
 ## Get a Fork ready
 
-You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
-Once you get a fork, you will clone it with `git clone url` where `url` is the link to your fork.
+1. Fork it: You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
+2. Once you get a fork, you will clone it with:
 
-Once that is done, you can start contributing by checking our [issues](https://github.com/FreeCodeCamp/wiki/issues) and creating nee pull requests.
+  ```sh
+  git clone https://github.com/YOUR-USERNAME/wiki.git
+  ```
 
-## Contribution Guides
+3. Add the remote: You will need a way to keep your fork syncronized with the original. using remotes is the right way.
```

Comment:
~~using remotes is the right way.~~ Using remotes is the right way.

"Using remotes is the right way.". Maybe too informal?